tsconfig.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
We want this at root?

Reply:
We need it for the typedoc, it also is a good practice to have one shared root tsconfig which the rest of our projects extend from, but that's a task for another PR